### PR TITLE
Bear.py: Maintainers are authors by default

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: coalib/bears/Bear.py [hunk 1]
```diff
@@ -136,6 +146,23 @@ def can_detect(cls):
...
+        """
+        :return: A set containing MAINTAINERS if specified, else takes
+                 AUTHORS by default.
+        """
+        return cls.AUTHORS if cls.MAINTAINERS == set() else cls.MAINTAINERS
+
+    @classproperty
+    def maintainers_emails(cls):
+        """
+        :return: A set containing MAINTAINERS_EMAILS if specified, else takes
```
Comment (hypothesist) [inline coalib/bears/Bear.py]: Double backticks

File: coalib/bears/Bear.py [hunk 2]
```diff
@@ -136,6 +146,23 @@ def can_detect(cls):
         """
         return cls.CAN_DETECT | cls.CAN_FIX
 
+    @classproperty
+    def maintainers(cls):
+        """
+        :return: A set containing MAINTAINERS if specified, else takes
```
Comment (hypothesist) [inline coalib/bears/Bear.py]: Same here

File: coalib/bears/Bear.py [hunk 3]
```diff
@@ -136,6 +146,23 @@ def can_detect(cls):
...
+                 AUTHORS by default.
+        """
+        return cls.AUTHORS if cls.MAINTAINERS == set() else cls.MAINTAINERS
```
Comment (hypothesist) [inline coalib/bears/Bear.py]: I'm wondering if maintainers should be authors + maintainers (just in case they are different)

Not sure.

Comment (abhsag24) [inline coalib/bears/Bear.py]: wondering the same thing

**Comment by abhsag24** [comment]
ack 0be6481

**Comment by hypothesist** [comment]
unack https://github.com/coala-analyzer/coala/pull/2353/commits/e19e6a4459d5e2d6c46c6f54f67458c38fd3ebbc (backticks and the other thing)

**Comment by Adrianzatreanu** [comment]
what codecov message?

**Comment by hypothesist** [comment]
@Adrianzatreanu `99.97% (+<.01%) compared to 9b22b21`

**Comment by Adrianzatreanu** [comment]
eh its like that all the time, not sure where we miss coverage on coala but it seems we do

File: coalib/bears/Bear.py (hunk 3 above)
Comment (Adrianzatreanu) [inline coalib/bears/Bear.py]: i dont think they should be the same. Imagine the case where I create a bear today. me and my 3 friends are maintaining it. in a month i quit the project. 1 year later, even though im the author, im not still a maintainer, right?

**Comment by Adrianzatreanu** [comment]
reack cf3d7db

**Comment by abhsag24** [comment]
ack e549ecd

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**